src/main/kotlin/com/msg/gcms/domain/webhook/util/CreateMessageUtil.kt

```diff
@@ -0,0 +1,7 @@
+package com.msg.gcms.domain.webhook.util
+
+import com.msg.gcms.domain.club.enums.ClubType
+
+interface CreateMessageUtil {
+    fun execute(club_name: String, club_type: ClubType, club_img: String): String
```

Comment:
```suggestion
    fun execute(clubName: String, clubType: ClubType, clubImg: String): String
```
앗 무지성 구현체에 suggestion 갈겨버렸네요